src/pywifes/wifes_wsol.py

```diff
@@ -879,7 +894,6 @@ def xcorr_shift_grid(slitlet_data, wave_guess, ref_interp):
     # pylab.plot(y_samp, all_stretches, 'bo')
     # pylab.plot(fval_y, stretch_fvals, 'g')
     # pylab.show()

```

Comment:
Is it useful the commented-out code above? If not, please remove it.

Reply:
Want to keep it. Probably will reused soon.